docs/install-1.6.md

```diff
@@ -165,3 +169,38 @@ kubectl config set-context service-catalog --cluster=service-catalog
 Note: Your cloud provider may require firewall rules to allow your traffic get in.
 Please refer to the [Troubleshooting](./walkthrough-1.6.md#troubleshooting) 
 section of the walkthrough document for details.
+
+## Troubleshooting
+
+### Non-default image installation
+
+The service catalog installation will by default use images of the controller
+and apiserver built at the latest tagged version. Should you be using a
+different state of the repo (for instance, the latest commit in the `master`
+branch), there may be conflicts with these images and the latest chart and
+walkthrough files. During installation, you may specify the images of the
+controller and apiserver to use. To do so, set your `REGISTRY` and `VERSION`
+environment variables to the appropriate container registry and image tag
+respectively and add the following flags to your `helm install` command:
+
+```console
+--set apiserver.image=${REGISTRY}apiserver:${VERSION} \
+--set controllerManager.image=${REGISTRY}controller-manager:${VERSION}
+```
+
+The `quay.io/kubernetes-service-catalog/` container registry stores images built
+from recent `master` commits. To use these images, set `REGISTRY` to point to
+this registry and `VERSION` to the abbreviated SHA of that commit via the
+following:
+
+```console
+REGISTRY=quay.io/kubernetes-service-catalog/
+VERSION=$(git describe --always --abbrev=7)
```

Comment:
maybe better to just put a sample SHA in here, and tell them that the `git` command can be handy for them if their local copy is on a known-deployed SHA

Reply:
Hmm... The point of the console commands is to let the user run them for intended behavior, not as an example. Is your worry that there's no example of what this version would actually look like? If so, how about I say `to the abbreviated SHA of that commit (for example, `a876fe3`)`?